test/Microsoft.SqlTools.ServiceLayer.Test/LanguageServer/PeekDefinitionTests.cs

```diff
@@ -184,6 +185,33 @@ public void GetSchemaFromDatabaseQualifiedNameWithInvalidNameTest()
             string actualSchemaName = peekDefinition.GetSchemaFromDatabaseQualifiedName(validDatabaseQualifiedName, objectName);
             Assert.Equal(actualSchemaName, expectedSchemaName);
         }
+
+        /// <summary>
+        /// Test Deletion of peek definition scripts for a valid temp folder that exists
+        /// </summary>
+        [Fact]
+        public void DeletePeekDefinitionScriptsTest()
+        {
+            PeekDefinition peekDefinition = new PeekDefinition(null);
+            var languageService = LanguageService.Instance;
+            Assert.True(Directory.Exists(FileUtils.PeekDefinitionTempFolder));
+            languageService.DeletePeekDefinitionScripts();
+            Assert.False(Directory.Exists(FileUtils.PeekDefinitionTempFolder));
+        }
+
+        /// <summary>
+        /// Test Deletion of peek definition scripts for a temp folder that does not exist
+        /// </summary>
+        [Fact]
+        public void DeletePeekDefinitionScriptsWhenFolderDoesNotExistTest()
+        {
+            var languageService = LanguageService.Instance;
+            PeekDefinition peekDefinition = new PeekDefinition(null);
+            FileUtils.SafeDirectoryDelete(FileUtils.PeekDefinitionTempFolder, true);
+            Assert.False(Directory.Exists(FileUtils.PeekDefinitionTempFolder));
+            // Expected not to throw any exception
+            languageService.DeletePeekDefinitionScripts();        
```

Comment:
just a suggestion, for future unit tests on this feature, we should try to create mock that will keep us from writing to the filesystem. Cleaning up after filesystem writes can be a huge pain for unit tests, if we can avoid it, let's do so.